src/components/App.js

```diff
@@ -1,28 +1,81 @@
 import React, { useState, useEffect } from 'react';
-
+import ReactDOM from "react-dom";
```

Comment:
I believe this import is unused and can be removed

Reply:
complete